pkg/utils/addresses.go

```diff
@@ -241,6 +251,7 @@ func addressesDefaultInternal(preferIPv6 bool, af AddressFilter, getAddrs addres
 			})
 		}
 	}
+	fmt.Println(addrs)
```

Comment:
Leftover debug statements? We typically use the log module for messages we want to keep.

Reply:
Yep, removing